setup.cfg

```diff
@@ -0,0 +1,54 @@
+[metadata]
+name = fparser
+author = Andrew Porter
```

Comment:
It must have been me that put this originally but it's not right. If possible, could you change this to "Pearu Peterson, Rupert Ford and Andrew Porter". Also, could you change the email address below to andrew.porter@stfc.ac.uk.

Reply:
Yep, taken from `setup.py`. Will update!